openassessment/xblock/staff_area_mixin.py

```diff
@@ -378,18 +378,49 @@ def get_student_info_path_and_context(self, student_username, expanded_view=None
         self_assessment = None
         peer_assessments = None
         submitted_assessments = None
+        peer_assessments_grade_context = []
+        self_assessment_grade_context = None
+        example_based_assessment_grade_context = None
+        staff_assessment_grade_context = None
```

Comment:
Would it be worthwhile to pair all these `XXX_assessment` and `XXX_assessment_context` item together in tuples?